Listening task (#66)
ALLUOS/ALLUOS.github.io: _posts/2021-03-01-listening.md

   tag: \#implementation
description: Learn more about the listening task.
image:
---

## Contents
1. Motivation
2. Task Flow
3. Materials
4. Implementation

<div id="motivation"></div>
## Motivation

> The first three tasks have been designed to provide practice with core language skills concerned with literacy – the ability to read, write and construct sentences in a foreign language. However, as Renukadevi (2014) suggests, complete language proficiency depends crucially on another skill, namely listening. Being able to comprehend spoken words is necessary for face-to-face communication, which is the most common end goal of studying a foreign language (eg. Morley, 2001) . Not only is it necessary in order to come up with an appropriate response, but importantly it also helps students to acquire the correct pronunciation, word stress and a more confident use of vocabulary and syntax, which all improve the students’ speaking competencies. Yildirim and Yildirim (2016) as well as others have pointed out that ever since school curriculums have transferred to a more communicative approach of foreign language teaching, listening skills have also been gaining in popularity in school environments. For this reason, we decided to create an additional task that can provide users with practice and more exposure to spoken English.

Reviewer: ```suggestion
The first three tasks have been designed to provide practice with core language skills concerned with literacy – the ability to read, write and construct sentences in a foreign language. However, as Renukadevi (2014) suggests, complete language proficiency depends crucially on another skill, namely listening. Being able to comprehend spoken words is necessary for face-to-face communication, which is the most common end goal of studying a foreign language (e. g., Morley, 2001). Not only is it necessary in order to come up with an appropriate response, but, importantly, it also helps students to acquire the correct pronunciation, word stress and a more confident use of vocabulary and syntax, which all improve the students’ speaking competencies. Yildirim and Yildirim (2016) as well as others have pointed out that ever since school curricula have transferred to a more communicative approach of foreign language teaching, listening skills have also been gaining in popularity in school environments. For this reason, we decided to create an additional task that can provide users with practice and more exposure to spoken English.
```

Reviewer: Sorry about the incorrect review formatting. I just fixed minor typos :)

However, because of the Telegram chosen as a platform for the Escapeling, we have encountered a series of difficulties while designing this task. 

First of all, in the typical classroom scenario the list of questions is presented to the students before the audio task, which in turn is usually played twice. In case of the Telegram, it is impossible to control the amount of times users get exposed to the audio, since we did not manage to come up with a tool that can get the audio file deleted from the chat after a certain amount of times it was played or certain amount of time that has passed after the file was sent to the players. Thus, a new scenario had to be created in such a way, that it both satisfies the learning goals of improving the users' listening skills and seems to be suitable for its implementation in the Telegram. Hence, we concluded that presenting the audio before the questions was the most suitable solution implementation-wise as well as the means to get players interested in the task and better engaged.

<div id="materials"></div>
## Materials

> At the moment, the audio database contains 45 recordings, each centered around a certain topic and labelled according to difficulty level. All files come from publicly available websites which are intended for learners and teachers of English as a foreign language, namely www.listenaminute.com, www.elllo.org and www.esl-lounge.com. We made sure that all sources legally allow the use of their audio materials for non-commercial purposes.

Reviewer: ```suggestion
At the moment, the audio database contains 45 recordings, each centered around a certain topic and labelled according to a difficulty level. All files come from publicly available websites which are intended for learners and teachers of English as a foreign language, namely [www.listenaminute.com](www.listenaminute.com), [www.elllo.org](www.elllo.org) and [www.esl-lounge.com](www.esl-lounge.com). We made sure that all sources legally allow the use of their audio materials for non-commercial purposes.
```

 
Along these lines, to keep the overall length of the task to an appropriate time window even if the users decide to listen more times, we decided to standardize the length of every audio file to approximately one minute. Files of such length are also going to be much easier on the device memory storage of the users, which is certainly an added plus.
 
In order to provide a more authentic learning experience, we aimed to provide a good variety of speakers with regard to their voices and accents, as well as a wide range of everyday topics, for which intermediate learners are expected to have the required core vocabulary. Some examples include Health, Media, French Fries and Job Duties. Unfamiliar accents can severely hinder students’ abilities to understand the spoken words (Gilakjani & Sabouri, 2016), so only speakers of the most common native English accents are included in the final data set.
 

> Since the maximum number of players is 4, we have designed 4 questions per audio, each with 3 possible options to choose from where exactly one is correct. The focus of the questions is also diverse. Some require a general understanding of the speaker’s attitude about the topic, others ask for a more detailed-oriented listening or distinguishing between homonyms (equally or similarly sounding words) based on context.

Reviewer: ```suggestion
Since the maximum number of players is four, we have designed four questions per audio, each with three possible options to choose from where exactly one is correct. The focus of the questions is also diverse. Some require a general understanding of the speaker’s attitude about the topic, others ask for a more detail-oriented listening or distinguishing between homonyms (equally or similarly sounding words) based on context.
```

> In terms of the difficulty level, the main motivation for the division was to eventually accommodate the adaptive module, which assigns a specific question to the users based on their previous performance. There does not seem to be a unified official method for determining the difficulty level of a listening task. Therefore, our process was as follows: the content (in form of a textual transcript) was first evaluated based on the language complexity and vocabulary using a combination of automatic tools ([1], [2]) to indicate a CEFR level [3] of a text. In addition to this indication, the data was also subjectively evaluated for speech qualities such as tempo and clarity, and the final level of difficulty resulted from agreement between three judges.

Reviewer: ```suggestion
In terms of the difficulty level, the main motivation for the division was to eventually accommodate the adaptive module, which assigns a specific question to the users based on their previous performance. There does not seem to be a unified official method for determining the difficulty level of a listening task. Therefore, our process was as follows: the content (in form of a textual transcript) was first evaluated based on the language complexity and vocabulary using a combination of automatic tools ([1](http://www.roadtogrammar.com/textanalysis/), [2](https://cefr.duolingo.com)) to indicate a CEFR level [3] of a text. In addition to this indication, the data was also subjectively evaluated for speech qualities such as tempo and clarity, and the final level of difficulty resulted from agreement between three judges.
```

 
Although our target audience is mainly B1-B2 students, since the intermediate proficiency group tends to be quite heterogeneous in terms of the individuals’ strengths and weaknesses, we also included slightly simpler audios that could theoretically be used with motivated A2 students, too.
 

> Each recording was carefully reviewed by at least three people, checking for the sound quality of the recording, correctness of the language and multiple choice questions, flagging of potentially sensitive subjects or unsuitable language, and as mentioned, the indicated level of difficulty.

Reviewer: ```suggestion
Each recording was carefully reviewed by at least three people, checking for the sound quality of the recording, correctness of the language and multiple choice questions, flagging potentially sensitive subjects or unsuitable language, and as mentioned, checking the indicated level of difficulty.
```
